src/CabanaPD_Solver.hpp

```diff
@@ -294,15 +279,21 @@ class SolverElastic
             log( std::cout, step, "/", num_steps, " ", std::scientific,
                  std::setprecision( 2 ), step * dt );
 
-            total_time = total_timer.seconds();
-            double rate = 1.0 * particles->n_global * output_frequency /
-                          ( total_time - last_time );
+            double step_time = _step_timer.time();
```

Comment:
Why "_step_timer" has the underscore symbol at the beginning but this is not used in the lines below (except _total_time)?

Reply:
Underscore denotes a member variable (but is style only, to be clear). These other variables are local scope only